otherlibs/stdune/test/bytes_unit_tests.ml

```diff
@@ -49,5 +52,17 @@ let%expect_test _ =
       1.23GB
       12.35GB
       123.46GB
-      1.23TB |}]
+      1.23TB
+      12.35TB
+      123.46TB
+      1234.57TB |}]
+;;
+
+(* Negative units get truncated but still printed as a negative. *)
+let%expect_test "Negative units" =
```

Comment:
Thanks for adding this test but I find the behaviour surprising. I think it would be better to just raise `Invalid_argument` or something like this on negative number of bytes.

Reply:
(So, maybe let's add a TODO to fix the current behaviour in future?)

Reply:
Would it be ok for me to do that in a follow up PR rather than this one?

Reply:
I think adding a TODO is better in this PR (because it adds a test whose behaviour seems surprising). Addressing the TODO should better be done separately.

Reply:
Sorry, my comment was for your first comment not you second. I've added a CR now.

Reply:
Ah, got it!